articles/user-guide/index.md

```diff
@@ -48,13 +48,13 @@ In the different topics of this guide, we detail the core concepts of the Q# lan
 
 ### Quantum Simulators and Resource Estimators
 
-- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general execution model between host programs and target machines.
+- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general run model between host programs and target machines.
```

Comment:
I think ["execution model"](https://en.wikipedia.org/wiki/Execution_model) communicates a concept that "run model" doesn't.